subcerts.go

```diff
@@ -154,7 +170,10 @@ func unmarshalCredential(serialized []byte) (*credential, error) {
 // unmarshalSubjectPublicKeyInfo parses a DER encoded SubjectPublicKeyInfo
 // structure into a public key and its corresponding algorithm.
 func unmarshalSubjectPublicKeyInfo(serialized []byte) (crypto.PublicKey, SignatureScheme, error) {
-	publicKey, err := x509.ParsePKIXPublicKey(serialized)
+	var publicKey crypto.PublicKey
```

Comment:
this change doesn't seem to bring anything

Reply:
Done.